MediaPlayer/MediaControls.js

```diff
@@ -248,6 +322,16 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {	// eslint-disable-line
 			super(props);
 		}
 
+		handleLoopButtonClick = (ev) => {
+			forward('onLoopButtonClick', ev, this.props);
```

Comment:
Could we remove the following code?
```
forward('onLoopButtonClick', ev, this.props);
forward('onNextButtonClick', ev, this.props);
forward('onPlayButtonClick', ev, this.props);
forward('onPreviousButtonClick', ev, this.props);
forward('onShuffleButtonClick', ev, this.props);
```

Because they are not defined in the `propTypes`. So those props won't be pass to these handlers.
Please correct me if I'm wrong.

If it is true, then we could bypass these handlers to Wrapped component. For example,
- remove `handleLoopButtonClick` function
- remove `delete props.onLoopChange;` in render()
- then `onLoopChange` prop will be passed to Wrapped component.